scripts/template_upload_ec2.py

```diff
@@ -224,13 +246,15 @@ def run(**kwargs):
          **kwargs: Kwargs are passed by template_upload_all.
      """
     mgmt_sys = cfme_data['management_systems']
-    for provider in list_provider_keys('ec2'):
+    if 'ec2east' in mgmt_sys: 
```

Comment:
This change came from the original review - but needs fixing because we need provider set.